nemo/collections/asr/metrics/wer.py

```diff
@@ -12,13 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import List
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, is_dataclass
+from typing import Callable, Dict, List, Optional, Union
 
 import editdistance
+import numpy as np
 import torch
+from omegaconf import DictConfig, OmegaConf
 from torchmetrics import Metric
 
-from nemo.collections.asr.parts.utils.rnnt_utils import Hypothesis
+from nemo.collections.asr.parts.submodules import ctc_greed_decoding
```

Comment:
Shouldn't the name be ctc_greedy_decoding?

Reply:
Good catch !